### PR TITLE
READY : Wrote test routine `startMinimalOptionThrowingExitCode`

### PR DESCRIPTION
**Pull request by JackieJoo**

### CODE REVIEW

**Comment by Wandalen** [comment]
Bad

**Comment by JackieJoo** [comment]
> Bad

Fixed

File: proto/wtools/abase/l4_process.test/Execution.test.s [hunk 1]
```diff
@@ -27185,6 +27185,238 @@ function startMultipleOptionProcedure( test )
...
+      .then( ( op ) =>
+      {
+        test.identical( op.exitCode, 0 );
+        test.identical( op.ended, true );
+        test.true( _.strHas( op.output, 'Process returned exit code' ) );
+        test.true( _.strHas( op.output, 'Launched as' ) );
+        test.true( _.strHas( op.output, 'Stderr' ) );
+        test.true( _.strHas( op.output, 'randomText is not defined' ) );
+        test.true( !_.strHas( op.output, 'stack' ) );
+
```
Comment (Wandalen) [inline proto/wtools/abase/l4_process.test/Execution.test.s:27355]: check whole output, not fragments

Comment (JackieJoo) [reply proto/wtools/abase/l4_process.test/Execution.test.s]: Done

File: proto/wtools/abase/l4_process.test/Execution.test.s [hunk 2]
```diff
@@ -26531,6 +26531,281 @@ function startMultipleOptionProcedure( test )
...
+        execPath : 'node ' + testAppParentPath,
+        outputCollecting : 1,
+      }
+
+      return _.process.startMinimal( options )
+      .then( ( op ) =>
+      {
+        test.identical( op.exitCode, 0 );
+        test.identical( op.ended, true );
+        let exp = `Process returned exit code 1\n`;
```
Comment (Wandalen) [inline proto/wtools/abase/l4_process.test/Execution.test.s:26697]: use multiline 
`
...
`
comments instead

File: proto/wtools/abase/l4_process.test/Execution.test.s [hunk 3]
```diff
@@ -26531,6 +26531,281 @@ function startMultipleOptionProcedure( test )
...
+    {
+      console.log( err );
+      return null;
+    })
+
+  }
+
+  function testApp()
+  {
+    class MyError extends Error
```
Comment (Wandalen) [inline proto/wtools/abase/l4_process.test/Execution.test.s:26795]: is it necessary?

Comment (JackieJoo) [reply proto/wtools/abase/l4_process.test/Execution.test.s:26795]: In order to check full output on all systems need to create a custom error with anticipated err.message ( set stack to null ).
If use `throw 'Text'` it causes eslint error - 'no-throw-literal'.

File: proto/wtools/abase/l4_process.test/Execution.test.s (hunk 2 above)
Comment (JackieJoo) [reply proto/wtools/abase/l4_process.test/Execution.test.s:26697]: done

**Pull request merged**